v3/src/components/graph/components/inspector-panel/graph-measure-panel.tsx

```diff
@@ -0,0 +1,80 @@
+import React from "react"
+import { Box, Button, Checkbox, Flex, FormControl, useToast}from "@chakra-ui/react"
+import t from "../../../../utilities/translation/translate"
+import { IGraphModel } from "../../models/graph-model"
+import { InspectorPalette } from "../../../inspector-panel"
+import ValuesIcon from "../../../../assets/icons/icon-values.svg"
+
+import "./point-format-panel.scss"
+
+interface IProps {
+  graphModel: IGraphModel
+}
+
+export const GraphMeasurePalette = ({graphModel}: IProps) => {
```

Comment:
I just noticed that the component is `GraphMeasurePalette` while the files are `graph-measure-panel.*`. Seems like they should be consistent.